standalone-metastore/pom.xml

```diff
@@ -365,8 +365,8 @@
         <optional>true</optional>
       </dependency>
       <dependency>
-        <groupId>mysql</groupId>
-        <artifactId>mysql-connector-java</artifactId>
+        <groupId>com.mysql</groupId>
```

Comment:
After switching to a new library, are these still valid?
https://github.com/apache/hive/blob/master/druid-handler/pom.xml#L187
https://github.com/apache/hive/blob/master/packaging/src/main/assembly/bin.xml#L53